api/api/controllers/agents_controller.py

```diff
@@ -401,20 +403,36 @@ async def restart_agent(request, agent_id, pretty=False, wait_for_complete=False
     return web.json_response(data=data, status=200, dumps=prettify if pretty else dumps)
 
 
-async def put_upgrade_agent(request, agent_id, pretty=False, wait_for_complete=False, wpk_repo=None, version=None,
-                            use_http=False, force=False):
-    """Upgrade agent using a WPK file from online repository.
+async def put_upgrade_agents(request, agents_list='*', pretty=False, wait_for_complete=False, wpk_repo=None,
```

Comment:
Why '*' as default value for agents_list if the parameter does not accept all agents?

Reply:
* deleted, new default agents_list is None